src/utils/canAccessLocalStorage.ts

```diff
@@ -0,0 +1,8 @@
+export const canAccessLocalStorage = () => {
+  try {
+    localStorage.getItem('test');
```

Comment:
Does something like:

```
(typeof window.localStorage !== 'undefined')
```

work as well instead of always `get`ting a fake item from storage?

Reply:
i tried a variation of ^ the undefined check and `'localStorage' in window` as well... it seems that the only option is to catch the access error

Reply:
Ah okay, interesting.  In that case, do you think we should name the key for this something a bit specific to chroma, like `chroma-react-storage` or something instead of `test`?

Reply:
sure, the actual string doesn't matter as any kind of usage of localStorage will trigger the security exception